### ISSUE TITLE
aria-multiselectable should be supported in role=tablist

### ISSUE THREAD

**Issue by eitanmk** [evcohen/eslint-plugin-jsx-a11y#82]
Using the latest version of this plugin (2.2.0), I received the error when running eslint:

`The attribute aria-multiselectable is not supported by the role tablist`

I checked in the ARIA spec and it looks like `aria-multiselectable` [should be allowed](https://www.w3.org/TR/wai-aria/states_and_properties#aria-multiselectable) in a `tablist` role.

I think this is simply a matter of adding `ARIA-MULTISELECTABLE` to the `TABLIST` entry in https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/src/util/attributes/role.json#L2473-L2497

(If that's the case, happy to submit a PR.)

**Comment by evcohen** [comment]
oversight 😄 
thanks for catching - yep that's all you should have to do.

**Comment by evcohen** [comment]
Published in [v2.2.1](https://github.com/evcohen/eslint-plugin-jsx-a11y/releases/tag/2.2.1)

**Comment by eitanmk** [comment]
Thanks for the quick turnaround! Installed 2.2.1 and can confirm the warning is gone.
